src/config.c

```diff
@@ -3191,7 +3191,9 @@ standardConfig static_configs[] = {
     createTimeTConfig("repl-backlog-ttl", NULL, MODIFIABLE_CONFIG, 0, LONG_MAX, server.repl_backlog_time_limit, 60 * 60, INTEGER_CONFIG, NULL, NULL), /* Default: 1 hour */
     createOffTConfig("auto-aof-rewrite-min-size", NULL, MODIFIABLE_CONFIG, 0, LLONG_MAX, server.aof_rewrite_min_size, 64 * 1024 * 1024, MEMORY_CONFIG, NULL, NULL),
     createOffTConfig("loading-process-events-interval-bytes", NULL, MODIFIABLE_CONFIG | HIDDEN_CONFIG, 1024, INT_MAX, server.loading_process_events_interval_bytes, 1024 * 1024 * 2, INTEGER_CONFIG, NULL, NULL),
+    createBoolConfig("enable-debug-assert", NULL, MODIFIABLE_CONFIG, server.enable_debug_assert, 0, NULL, NULL),
```

Comment:
let's add a HIDDEN_CONFIG? I feel that this configuration item should not be exposed to users, that is, it should not be put into valkey.conf (or it should be a debug subcommand)

Reply:
make sense. will remove it from valkey.conf. 

> it should be a debug subcommand

I think a real (hidden) server config would give us coverage on the server initialization path (right after configs parsing) as well.

Reply:
Historically we've marked all configs as major decisions since it is a change to the user facing API. Users can toggle this config at runtime. By default, we recommend users should disable the `DEBUG` command. So, I think this should be marked as a static config so that nobody has the ability to enable this.